whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/DeltaFile.java

```diff
@@ -0,0 +1,52 @@
+package net.ripe.db.nrtm4.persist;
```

Comment:
I suggest using the package name `dao` for consistency with the rest of Whois

Reply:
and he class names too

Reply:
package renamed. not sure what you mean about the class names -- do you mean rename *Repository to *Dao? "Repository" is already used in Whois, and these classes are annotated "Repository", while DAO is architecturally on layer above the Repository. for my money "Repository" is both consistent and correct, so please confirm if you want them to be renamed to Dao.